openedx/features/course_experience/urls.py

```diff
@@ -22,6 +23,11 @@
         name='openedx.course_experience.course_updates',
     ),
     url(
+        r'^reviews$',
```

Comment:
@andy-armstrong Can you help us understand why bookmark is a top-level feature and updates is in here? We should probably document your thinking once it is clarified.